docs/fundamentals/flow/flow-api.md

```diff
@@ -2,7 +2,10 @@
 
 # Flow API
 
-Every `Flow` provides an API to receive requests over the network. Supported protocols are gRPC, HTTP and websockets. We recommend to use gRPC for the best performance, which is also the default protocol. Requests can be send via the Jina Python Client or separate clients like Curl.
+**Flow** ties Executors together into a processing pipeline, provides scalability and facilitates deployments in the cloud. Every `Flow` provides an API to receive requests over the network. Supported protocols are gRPC, HTTP and websockets. 
+
+We recommend to use gRPC for the best performance, which is also the default protocol. Requests can be send via the Jina Python Client or separate clients like Curl. Flow streamlines and scales Executors.
```

Comment:
```suggestion
We recommend to use gRPC for the best performance, which is also the default protocol. Requests can be sent via the Jina Python Client or separate clients like Curl. Flow streamlines and scales Executors.
```

Reply:
This sentence is weird. 

You define Flow, then add a warning and then keep adding a definition sentence?

Reply:
Also, more than recommending grpc, we should explain the caveats of http

Reply:
merge first , fix it later